.travis.yml

```diff
@@ -14,8 +14,8 @@ env:
     - OPENSSL=1.1.1c
     - OPENSSL_DIR="$HOME/multissl/openssl/${OPENSSL}"
     - PATH="${OPENSSL_DIR}/bin:$PATH"
-    # Use -O3 because we don't use debugger on Travis-CI
-    - CFLAGS="-I${OPENSSL_DIR}/include -O3"
```

Comment:
Please don't change here.  This environment variable is used for all build, including document test.

Reply:
So what would you suggest instead?

Note that this affects only the Linux Travis CI build. On other platforms and with other CI services, we still use `-O3`.

Reply:
> So what would you suggest instead?

Just remove '-O3`.

> Note that this affects only the Linux Travis CI build. 

Affects *all* of Linux Travis CI build.

Reply:
https://travis-ci.org/python/cpython/jobs/551191274

Actually, it affects even macOS build.